commit/chainfee/outcome.go

```diff
@@ -96,7 +96,7 @@ func (p *processor) getConsensusObservation(
 ) (Observation, error) {
 	aggObs := aggregateObservations(aos)
 
-	// consensus on the fChain map uses the role DON F value
+	// consensus on the fChain map uses the role DON fRoleDon value
```

Comment:
The comment already says "the role DON F value", changing it to "the role DON fRoleDon value" seems weird.

Reply:
That was a replace all typo :D